### PR TITLE
Add revocation list endpoint

### PR DESCRIPTION
**Pull request by OR13**
in our platform we host these credential at a different endpoint:

```
/credentials/{guid}
```

However if would be easy for us to update these endpoints to match whatever we agree is required for revocation interop here.

As an issuer you do need to host these files somewhere, (or trust someone else to host them and update when their status changes).

This is to start that conversation, because changing a credential status does not actually work, unless you define how to get the credential revocation list.

I updated the PR to match what we have today.

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

File: docs/components/CredentialLinkedDataProof.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ttile: Credential Linked Data Proof
```
Comment (brownoxford) [inline docs/components/CredentialLinkedDataProof.yml:1]:
```suggestion
title: Credential Linked Data Proof
```

File: docs/issuer-service.yml [hunk 2]
```diff
@@ -36,6 +36,25 @@ paths:
           description: invalid input!
         "500":
           description: error!
+
+  /credentials/{id}:
```
Comment (nissimsan) [inline docs/issuer-service.yml:40]: Is this the right name for it? Shouldn't it explicitly indicate revocation?

Comment (OR13) [reply docs/issuer-service.yml:40]: see our current api for why it is the way it is here.

Comment (OR13) [reply docs/issuer-service.yml:40]: We could of course change our API. 

But I would prefer to only do that if we need to based on what the spec here says.

**Comment by OR13** [comment]
This PR has been over taken by https://github.com/w3c-ccg/traceability-interop/pull/69

**Pull request closed without merging**